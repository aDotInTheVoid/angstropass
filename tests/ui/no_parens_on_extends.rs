#[nanopass::langs]
mod langs {
    #[extends]
    mod baz {}
}

fn main() {}
