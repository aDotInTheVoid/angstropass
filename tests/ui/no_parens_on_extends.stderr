error: expected attribute arguments in parentheses: #[extends(...)]
 --> tests/ui/no_parens_on_extends.rs:3:7
  |
3 |     #[extends]
  |       ^^^^^^^
